### DISCUSSION TITLE
Adaptative Cruise Control Clio 5

### DISCUSSION THREAD

**Discussion by ALBALOCHI** [cedricp/ddt4all#1039] [Q&A]
Hello 
I have a Clio 5 2019 1.0 TCE equipped with front camera and radar so I already have the functions like Distance Warning (DW). 
I belive that the car is most likly adapted to activate the ACC by changing the steering wheel with one has the ACC fucntion. 
after a small research I found that the related paramters are in sertain ECUs like 
- ABS 
- FCAM "Front Camera"
- FRRAD "Front Radar" 

I injected the ABS with new software version which is coming from a Clio 5 with ACC from the factory, but I still can't have the ACC function. 
Does anyone had already succeeded to activate this fuction ? maybe we could dicuss the possibility. 
Thank you . 

![image](https://user-images.githubusercontent.com/47195112/191088002-04be64f2-f779-4e13-8f89-d79633003914.png)

![image](https://user-images.githubusercontent.com/47195112/191087797-47cc9cda-00bf-4f72-9110-27d21d9a5088.png)

**Comment by Erkanaga** [comment]
Bende deniyorum tuşlu direksiyon buldum taktım k.k tuşları hatası verdi kohm ları enjeksiyon beyni kabul etmiyor uğraşıyorum bir sonuç bulursam paylaşırım

**Comment by ALBALOCHI** [comment]
UP

**Reply by dawnwhitewater** [reply]
Up

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/836

**Comment by dawnwhitewater** [comment]
Up

**Comment by dawnwhitewater** [comment]
> Hello 
> 
> I have a Clio 5 2019 1.0 TCE equipped with front camera and radar so I already have the functions like Distance Warning (DW). 
> 
> I belive that the car is most likly adapted to activate the ACC by changing the steering wheel with one has the ACC fucntion. 
> 
> after a small research I found that the related paramters are in sertain ECUs like 
> 
> - ABS 
> 
> - FCAM "Front Camera"
> 
> - FRRAD "Front Radar" 
> 
> 
> 
> I injected the ABS with new software version which is coming from a Clio 5 with ACC from the factory, but I still can't have the ACC function. 
> 
> Does anyone had already succeeded to activate this fuction ? maybe we could dicuss the possibility. 
> 
> Thank you . 
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/47195112/191088002-04be64f2-f779-4e13-8f89-d79633003914.png)
> 
> 
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/47195112/191087797-47cc9cda-00bf-4f72-9110-27d21d9a5088.png)
> 
> 
> 
> 

Do you have any improvement bro.
Maybe we can talk for that 
capoeira_35@hotmail.com

**Reply by Erkanaga** [reply]
Acc must be active in the injection brain. My vehicle is Rs line 1.3. If I find a device to capture the flash information of the brain, I will remove it from the talisman.

**Reply by ucari3** [reply]
ucar324@gmail.com

**Comment by dawnwhitewater** [comment]
Up

**Comment by muchecomacdonald** [comment]
Up

**Comment by ernestzz** [comment]
yes, in Megane IV 2022, Arkana. :)

**Reply by xelox9** [reply]
can you explain how you proceed ?
thank you

**Reply by talhaturann** [reply]
bunu bende merak ediyorum retrofit olarak uygulayacağım
